specs/spec-upload-fragment.md

```
@@ -0,0 +1,808 @@
+  exposed in URLs.
+
+### 6.3 Encoding format
+
+Encode each part with a type prefix, then join with `.`:
+
+- String part: `s~<base64url(utf8)>`
+- Number part: `n~<decimal>`
+
+Example:
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)